www/includes/easyparliament/templates/html/search/by-person.php

```diff
@@ -0,0 +1,108 @@
+                        else if ( $house == 3 ) { ?>No results for MLAs only<?php } ?> |
+                        <a href="<?= $this_url->generate('html') ?>">Show results for all speakers</a></li>
+                </ul>
+                <?php } else { ?>
+                <p class="search-results-legend"><?= $error ?></p>
+                <?php } ?>
+            <?php } ?>
+
+            <?php if ( $wtt ) { ?>
+                <p><strong>Now, try reading what a couple of these Lords are saying,
+                to help you find someone appropriate. When you've found someone,
+                hit the "I want to write to this Lord" button on their results page
+                to go back to WriteToThem.
+                </strong></p>
+            <?php } ?>
+
+            <?php if ( isset($speakers) && count($speakers) ) { ?>
+                <?php if ( !$wtt ) { ?>
+                <ul class="search-result-display-options">
+                    <li>Results grouped by person</li>
+                    <li><?php if ( $house == 0 ) { ?>Show All<?php } else { ?><a href="<?= $this_url->generate('html') ?>">Show All</a><?php } ?> |
+                        <?php if ( $house == 1 ) { ?>MPs only<?php } else { ?><a href="<?= $this_url->generate('html', array('house'=>1)) ?>">MPs only</a><?php } ?> |
+                        <?php if ( $house == 2 ) { ?>Peers only<?php } else { ?><a href="<?= $this_url->generate('html', array('house'=>2)) ?>">Lords only</a><?php } ?> |
+                        <?php if ( $house == 4 ) { ?>MSPs only<?php } else { ?><a href="<?= $this_url->generate('html', array('house'=>4)) ?>">MSPs only</a><?php } ?> |
+                        <?php if ( $house == 3 ) { ?>MLAs only<?php } else { ?><a href="<?= $this_url->generate('html', array('house'=>3)) ?>">MLAs only</a><?php } ?></li>
+                    <li><a href="<?= $ungrouped_url->generate() ?>">Ungroup results</a></li>
+                </ul>
+
+                <p class="search-results-legend">The <?= isset($limit_reached) ? '5000 ' : '' ?>most recent mentions of the exact phrase <em class="current-search-term"><?= $searchstring ?></em>, grouped by speaker name.</p>
+                <?php } ?>
+
+                <table class="search-results-grouped">
+                    <thead>
+                        <tr>
+                            <th>Occurences</th>
+                            <th>Speaker</th>
+                            <th>Date range</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    <?php foreach ( $speakers as $pid => $speaker ) { ?>
+                        <?php if ( $wtt && $pid == 0 ) { continue; } // skip heading count for WTT lords list ?>
+                        <tr>
+                            <td><?= $speaker['count'] ?></td>
+                            <td>
+                                <?php if ( $pid ) { ?>
```

Comment:
This will make the Lord name a link even if the Lord is no longer active, it's not identical to the old code.